fabric-dimensions-v1/build.gradle

```diff
@@ -9,5 +9,6 @@ dependencies {
 moduleDependencies(project, [
 		'fabric-api-base',
 		'fabric-networking-v0',
-		'fabric-registry-sync-v0'
+		'fabric-registry-sync-v0',
+		'fabric-lifecycle-events-v1'
```

Comment:
Please place the lifecycle events dependencies under the `testmodCompile` configuration?

That is because only the testmod depends on the lifecycle events.